ui/src/StateContainer.js

```diff
@@ -132,14 +162,20 @@ export class StateContainer extends LitElement {
             @skip-prompt="${this.handleChangePrompt}"
             @end-session="${this.handleEndSession}"
             @update-wave="${this.handleUpdateWave}"
+            @update-toast="${this.handleUpdateToast}"
         >
             <vox-view-container
                 .view=${this.view}
                 ?can-record=${this.canRecord}
                 ?is-recording=${this.isRecording}
                 .audio-stream=${this.audioStream}
                 .user=${this.user}
-            ></vox-view-container>
+                .toast=${this.toast}
+            >
+                <div slot="toast">
```

Comment:
Why is the toast rendered in a slot instead of as a sibling of vox-view-container?